src/components/AccountLoginForm.jsx

```diff
@@ -26,6 +26,7 @@ const AccountLoginForm = ({ t, isOAuth, customView, fields, error, dirty, submit
               name={inputName}
               placeholder={t('account.connection.login.password.placeholder')}
               invalid={!!error}
+              noPasswordSave
```

Comment:
I prefer noAutoFill in that case since autoFill is the default behavior in the browsers